frontend/src/components/Output/ModelDetailModal.js

```diff
@@ -5,17 +5,19 @@ import PropTypes from "prop-types";
 
 import InfoTable from "./InfoTable";
 import ModelOptionsTable from "./ModelOptionsTable";
-import BenchmarkDose from "./BenchmarkDose";
 import ModelData from "./ModelData";
 import ModelParameters from "./ModelParameters";
 import GoodnessFit from "./GoodnessFit";
 import CDFTable from "./CDFTable";
 import CDFPlot from "./CDFPlot";
-import CSLoglikelihoods from "./CSLoglikelihoods";
 import DoseResponsePlot from "../common/DoseResponsePlot";
 import CSTestofInterest from "./CSTestofInterest";
+import DichotomousSummary from "./Summary/DichotomousSummary";
 
 import * as dc from "../../constants/dataConstants";
+import DichotomousDeviance from "./AnalysisDeviance/DichotomousDeviance";
```

Comment:
no need to build a deeper nesting here for deviance; keep flat

Reply:
removed nested file system